packages/dotnet-sdk/src/TeamsFx/Configuration/AuthenticationOption.cs

```diff
@@ -27,6 +27,12 @@ public class AuthenticationOptions
     [Required(ErrorMessage = "Client secret is required")]
     public string ClientSecret { get; set; }
 
+    /// <summary>
+    /// Login page for Teams to redirect to.
+    /// </summary>
+    [RegularExpression(@"^http(s)?://[-a-zA-Z0-9@:%._\+~#=/]{1,100}$")]
```

Comment:
Is `100` the limitation of teams auth api?
According to [AAD doc](https://learn.microsoft.com/en-us/azure/active-directory/develop/reply-url#maximum-uri-length), the aad redirect url length limit is `256`.

Reply:
Updated, thanks!